### PR TITLE
Tweak emails

### PR DESCRIPTION
**Pull request by railsagainstignorance**

### CODE REVIEW

File: absorb.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ const generateS3PublicURL = require('./bin/lib/get-s3-public-url');
 
 const S3 = new AWS.S3();
 
+const ingestorAdminUrl = 'https://ftlabs-audio-rss.herokuapp.com/ft';
```
Comment (seanmtracey) [inline absorb.js]: This should be an environment variable

File: absorb.js [hunk 2]
```diff
@@ -133,7 +135,7 @@ function checkForData(){
 
 							if (err && err.code === 'NotFound') {
 								// We don't have that audio file, let's grab it
-								debug(`We don't have the audio for ${itemUUID}. Fetching from ${item.link}`);
+								debug(`We dont have the audio for ${itemUUID}. Fetching from ${item.link}`);
```
Comment (seanmtracey) [inline absorb.js:138]: Why do you hate `'`?

Comment (railsagainstignorance) [reply absorb.js:138]: because sublime

File: absorb.js (hunk 1 above)
Comment (railsagainstignorance) [reply absorb.js]: done

**Comment by seanmtracey** [comment]
👍

**Pull request merged**